src/patternfly/patternfly-charts.scss

```diff
@@ -50,3 +51,97 @@ $pf-chart-color-black-200: $pf-color-black-400;
 $pf-chart-color-black-300: $pf-color-black-500;
 $pf-chart-color-black-400: $pf-color-black-600;
 $pf-chart-color-black-500: $pf-color-black-700;
+
+
+// typography
+$pf-chart-global--FontSize: $pf-global--FontSize--sm;
```

Comment:
font should be 12px

Reply:
for donut charts with the a single label in the middle representing data value, should be 24px

Reply:
legend should be using 14px text